.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,6 @@
-Describe your pull request!
+Changes proposed in this pull request:
```

Comment:
I would like some spot to just paste a jira ticket for ease of use.

Reply:
What would you propose?

Reply:
```
Is there a Jira ticket associated with this PR? Link here:
```